.buildkite/pipeline.quick.yml

```diff
@@ -85,9 +85,9 @@ steps:
     concurrency_group: 'browserstack-app'
     concurrency_method: eager
 
-  - label: ':android: Android 9 NDK r21 smoke tests'
-    key: 'android-9-smoke'
-    depends_on: "fixture-r21"
+  - label: ':android: Android 9 NDK r21 end-to-end tests - batch 1'
```

Comment:
This can go in another PR/ticket as it's a future optimization, but is it worth discussing what device the quick step should run on?

IIRC Android 9 was picked because it was most popular at the time, but Android 11 is the most commonly used device right now - so perhaps it'd make sense to run by default on Android 11 (or 12). There's probably also a case for looking at Appium performance over a couple of CI runs, and leaning towards the fastest option.

Reply:
No objection to moving on from 9, I just don't think it's been explicitly considered until now.  Theissue we might have is availability when 11/12 are [tier 3/4 devices](https://www.browserstack.com/device-tiers) when Android 9 is tier 1.  So although the devices might be quicker, we could spend longer waiting for them (and potentially timing out).

Reply:
Got it. Sticking with tier 1 sounds the best option for reliability for now in that case 👍